ASP.NET API/WAVC_WebApi/Data/ApplicationDbContext.cs

```diff
@@ -12,13 +8,12 @@ public class ApplicationDbContext : IdentityDbContext
     {
         public new DbSet<ApplicationUser> Users { get; set; }
         public DbSet<Relationship> Relationships { get; set; }
-        public DbSet<FriendRequest> FriendRequests { get; set; }
 
         public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options)
         {
         }
 
-        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        /*protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
If it's not used delete it..

Reply:
I wanted to keep it in case you want to use something from it, but commented out because I had error with it.